### PR TITLE
Bug fix and log cleanup

### PR DESCRIPTION
**Pull request by shumonsharif**

### CODE REVIEW

**Comment by shumonsharif** [comment]
Woops just realized I forgot to request a review here too ... @famod

File: runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public WrapperHelper compile(Class<?> wrapperType, Method[] setMethods,
             Method[] getMethods, Method[] jaxbMethods,
             Field[] fields, Object objectFactory) {
         Logger LOG = LogUtils.getL7dLogger(Target_org_apache_cxf_jaxb_WrapperHelperClassGenerator.class);
-        LOG.info("compileWrapperHelper substitution");
+        LOG.fine("compileWrapperHelper substitution");
```
Comment (famod) [inline runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java:79]: I think `debug` is more common. But it's a matter of taste...

File: runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java [hunk 2]
```diff
@@ -100,7 +100,7 @@ public WrapperHelper compile(Class<?> wrapperType, Method[] setMethods,
                     try {
                         cls = Thread.currentThread().getContextClassLoader().loadClass(newClassName);
                     } catch (ClassNotFoundException e) {
-                        LOG.warning("Wrapper helper class not found : " + e.toString());
+                        LOG.fine("Wrapper helper class not found : " + e.toString());
```
Comment (famod) [inline runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java:103]: I missed that myself elsewhere (need to send a PR for that some day), but we should use the "formatting" variant wherever possible, meaning `.debugf("Wrapper helper class not found : %s", e)`.

File: runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java (hunk 1 above)
Comment (shumonsharif) [reply runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java:79]: This one is actually not a `org.jboss.logging.Logger`, instead it is [`java.util.logging.Logger`](https://docs.oracle.com/en/java/javase/11/docs/api/java.logging/java/util/logging/Logger.html) ... there is no debug or debugf method.

File: runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java (hunk 2 above)
Comment (shumonsharif) [reply runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java:103]: Agreed, I think we need a general PR to address much of the logging throughout the code; the intent with this PR was just the minor log level change.

File: runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java (hunk 1 above)
Comment (famod) [reply runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java:79]: Ah, good point.

**Pull request merged**